### DISCUSSION TITLE
Opening windows menu

### DISCUSSION THREAD

**Discussion by albinahlback** [i3/i3#5019] [Q&A] [answered]
Some applications require the use windows menu in order to fully utilize the program. I'm having trouble opening this menu. I have tried to search through the documentation, googling, and searching here, but I cannot find any solution for this.

How can I open the windows menu on i3? Do I need to set something in the configuration?

**Comment by Airblader** [comment, marked as answer]
What's "the windows menu"?

**Reply by albinahlback** [reply]
I think Firefox calls it "Menu bar" and for Firefox it can be displayed by clicking ALT. It can contain tabs such as "File", "View", "Tools", "Help", etc.

**Reply by Airblader** [reply]
That's not something the window manager is involved in. It's probably part of the toolkit the application uses (Gtk, Qt, ...).

**Reply by albinahlback** [reply]
Okay, thanks for answer!
